include/onnxruntime/core/providers/nv_tensorrt_rtx/nv_provider_options.h

```diff
@@ -35,6 +35,9 @@ constexpr const char* kCudaGraphEnable = "nv_cuda_graph_enable";
 constexpr const char* kONNXBytestream = "nv_onnx_bytestream";
 constexpr const char* kONNXBytestreamSize = "nv_onnx_bytestream_size";
 constexpr const char* kMultiProfileEnable = "nv_multi_profile_enable";
+constexpr const char* kUseExternalDataInitializer = "nv_use_external_data_initializer";
+constexpr const char* kExternalDataBytestream = "nv_external_data_bytestream";
+constexpr const char* kExternalDataBytestreamSize = "nv_external_data_bytestream_size";
```

Comment:
DO we really want to expose these options on a new EP ? We want to move to constant initializers as input very soon and I do not know what the deprecation policy on such provider options is for us. 

This is not something that we want to maintain IMHO.